deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -8,53 +8,37 @@
     time   ${[record["log"].split(/[\n\t]+/)[0]].map! {|item| JSON.parse(item)["time"]}.join("")}
   </record>
 </filter>
+# match all  container logs and label them @NORMAL
 <match containers.**>
   @type relabel
   @label @NORMAL
 </match>
 <label @NORMAL>
+  #  only match fluentd logs based on fluentd container name
+  {{ printf "<filter **%s**>" (include "sumologic.fullname" .) }}
```

Comment:
The filter plugin tag for matching fluentd logs is generated via the release name template, so in case user changes the release name , we would still match it.